doc/DESI_SPECTRO_DATA/NIGHT/EXPID/desi-EXPID.rst

```diff
@@ -16,138 +16,302 @@ There is one HDU per spectrograph camera with EXTNAMEs like
 B0, B1, ... R0, R1, ... Z8, Z9.  The structure of each of these is
 the same; only one is explicitly documented below.
 
-====== ======= ===== ===================
-Number EXTNAME Type  Contents
-====== ======= ===== ===================
-HDU0_  PRIMARY IMAGE Blank except for header keywords
-HDU1_  B0      IMAGE Raw data from the b0 spectrograph
-====== ======= ===== ===================
+================= ========= ======== ====================================
+Number            EXTNAME   Type     Contents
+================= ========= ======== ====================================
+HDU0_             SP0       IMAGE    Blank except for header keywords
+HDU1_             SPECTCON0 BINTABLE Telemetry data
+HDU2_             B0        IMAGE    Raw data from the b0 spectrograph
+`HDU3 -- HDU31`_  various   IMAGE    Raw data similar to b0 spectrograph.
+================= ========= ======== ====================================
 
 FITS Header Units
 =================
 
 HDU0
 ----
 
-EXTNAME = PRIMARY
+EXTNAME = SP0
 
-Blank except for header keywords
+Blank except for header keywords.
+
+Keywords marked **MISSING** were listed in a previous version of this file,
+but are not present in the spectrograph functional verification test "sp0" files.
```

Comment:
Thanks for flagging these as **MISSING** and not just removing them, so that we can review them on a case by case basis for whether the actual files or the datamodel should be updated.